Formula/kubeval.rb

```diff
@@ -0,0 +1,32 @@
+class Kubeval < Formula
+  desc "Validate Kubernetes configuration files, supports multiple Kubernetes versions"
+  homepage "https://www.kubeval.com/"
+  url "https://github.com/instrumenta/kubeval/archive/v0.16.1.tar.gz"
+  sha256 "a6a8a34d58af25ff111ad68c2781128a05960a04ee27ca8ff39dc20f369f854e"
+  license "Apache-2.0"
+  head "https://github.com/instrumenta/kubeval.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.commit=#{tap.user.downcase}
+      -X main.date=#{time.iso8601}
+    ].join(" ")
+    system "go", "build", *std_go_args(ldflags: ldflags)
+
+    pkgshare.install "fixtures"
```

Comment:
If those are only used for our test then I would say we should create a resource for this purpose.

Reply:
Yeah, that also works for me, but in general, I do like to have fixtures as examples in the `pkgshare`. What do you think?

Reply:
Okay I see some formulae with logic similar to this.